lib/spring/test/acceptance_test.rb

```diff
@@ -83,14 +83,14 @@ def without_gem(name)
       end
 
       test "help message when called without arguments" do
-        assert_success "bin/spring", stdout: 'Usage: spring COMMAND [ARGS]'
+        assert_success "bundle exec bin/spring", stdout: 'Usage: spring COMMAND [ARGS]'
```

Comment:
@grosser Do you really want me to change it to `with_clean_env`?
Actually, it needs just 2 lines of change.